components/telemetry-operator/internal/fluentbit/plugin_validator.go

```diff
@@ -90,3 +94,26 @@ func getSectionName(section map[string]string) (string, error) {
 	}
 	return "", fmt.Errorf("configuration section does not have name attribute")
 }
+
+func getMatchCondition(section map[string]string) string {
+	if matchCond, hasKey := section["match"]; hasKey {
+		return matchCond
+	}
+	return ""
+}
+
+func isMatchCondValid(pluginName, matchCond, logPipelineName string, logPipelines *telemetryv1alpha1.LogPipelineList) (bool, []string) {
+	if strings.Contains(matchCond, fmt.Sprintf("%s.", logPipelineName)) {
```

Comment:
Is it intended that we check for the current logPipelineName for `%s.` and for other logpipelines for `%s.*`

Reply:
good catch fixed.